### PR TITLE
iframe clipboard fix

### PR DESCRIPTION
**Pull request by ignapas**
## What do these changes do?

Adds the ``allow`` attribute to the ``iframe`` element to gove the services access to the host's clipboard.
## Related issue/s
https://git.speag.com/oSparc/osparc-s4l/-/issues/410

## How to test

* Open S4L
* Go to logger
* Hit 'Copy to clipboard'
* Ctrl+V somewhere
* Hopefully some logs are pasted

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]

File: services/docker-compose.local.yml [hunk 1]
```diff
@@ -146,25 +146,25 @@ services:
...
+  #     - target: 80
+  #       published: 9081
+  #     - target: 10081
+  #       published: 10081
+  #     - target: 8080
+  #       published: 8080
+  #   deploy:
+  #     labels:
+  #       # traefik dashboard UI accessible through http://127.0.0.1:8080/dashboard/
+  #       # traefik REST API accessible through for example http://127.0.0.1:8080/api/http/routers
```
Comment (sanderegg) [inline services/docker-compose.local.yml:160]: hmm, this does not look good, are you sure about this comments?

**Review by sanderegg — CHANGES_REQUESTED** [review]
not sure you want to push the change to docker-compose.local.yml

File: services/docker-compose.local.yml (hunk 1 above)
Comment (ignapas) [reply services/docker-compose.local.yml:160]: lol... thanks Sylvain, nice catch. no that we had to do to try testing it locally...

**Review by sanderegg — APPROVED** [review]
👍

**Pull request merged**